crates/matrix-sdk/CHANGELOG.md

```diff
@@ -25,6 +25,10 @@ All notable changes to this project will be documented in this file.
 - `Room::set_unread_flag()` now sets the stable `m.marked_unread` room account data, which was
   stabilized in Matrix 1.12. `Room::is_marked_unread()` also ignores the unstable
   `com.famedly.marked_unread` room account data if the stable variant is present.
+- `Encryption::encrypt_and_send_raw_to_device`: Introduced as an experimental method for
+  sending custom encrypted to-device events. This feature is gated behind the
+  `experimental-send-custom-to-device` flag, as it remains under active development and may undergo changes.
+
```

Comment:
Please add a link to the PR.

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/4998/commits/230c9581589f71317d9c3c6462359818ff33a96d